qiita_db/base.py

```diff
@@ -151,6 +177,10 @@ def __init__(self, id_):
         if not self._check_id(id_):
             raise QiitaDBUnknownIDError(id_, self._table)
 
+        if not self._check_portal(id_):
+            raise QiitaDBError("Inaccessible in current portal: %s" %
```

Comment:
`"%s with id %d inaccessible in current portal: %s" % (self.__class__.__name__, id_, qiita_config.portal)`
It can be useful while looking at logs.